webapp/channels/src/components/user_settings/advanced/performance_debugging_section/index.ts

```diff
@@ -1,26 +1,37 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
```

Comment:
The autofix got a hiccup here, it seems :stuck_out_tongue:

Reply:
Done.